silnlp/common/utils.py

```diff
@@ -50,6 +50,10 @@ def merge_dict(dict1: dict, dict2: dict) -> dict:
             dict1[key] = value
     return dict1
 
+def ulist(seq: list):
```

Comment:
Can you give this a more descriptive name? It is not clear what its purpose is. It would also be helpful to add type hints.

Reply:
unique lists - keep first element.  Does the same thing as a OrderedDict, but a little more explicitly and quickly.

Reply:
Updated definition and added comment.